File: Beginner/06. Java/BubbleSort.java
import java.util.*;

public class BubbleSort {
  public static void main(String[] args)
  {
    Scanner scn = new Scanner(System.in);
    Random r = new Random();

    //Input size of Array
    int n = scn.nextInt();
    int[] arr = new int[n];

    //Random values for Array
    for (int i = 0; i < n; i++)
      arr[i] = r.nextInt(n);

    //Bubble Sort Algo
    for (int i = 0; i <= n - 2; i++)
    {
      for (int j = 0; j <= n - 2 - i; j++)
      {
        if (arr[j] > arr[j + 1])
        {
          int temp = arr[j];
          arr[j] = arr[j + 1];
          arr[j + 1] = temp;
        }
      }
    }

    //Print sorted array
    for(int i : arr)
      System.out.print(i + "\t");
  }
}

File: Beginner/03. Python/bubble_sort.py
#Bubble sort
def bubbleSort(arr): 
    n = len(arr) 
    for i in range(n-1): 
        for j in range(0, n-i-1): 
            if arr[j] > arr[j+1] : 
                arr[j], arr[j+1] = arr[j+1], arr[j] 
  
arr = [74, 36, 15, 92, 10, 47, 82] 
bubbleSort(arr) 
print ("Sorted array is:") 
for i in arr: 
    print (i,end=" ")
